include/avif/avif.h

```diff
@@ -1160,6 +1160,11 @@ typedef struct avifDecoder
     // Can be useful to decode the gain map image only.
     avifBool ignoreColorAndAlpha;
 #endif
+
+    // This is true when avifDecoderParse() detects an animation track in the image. If this is true, the image can be decoded
```

Comment:
Nit: It would be better to consistently use the term "image sequence" for animations in this header. For example, this is how the decoder source is documented:

```
typedef enum avifDecoderSource
{
    // Honor the major brand signaled in the beginning of the file to pick between an AVIF sequence
    // ('avis', tracks-based) or a single image ('avif', item-based). If the major brand is neither
    // of these, prefer the AVIF sequence ('avis', tracks-based), if present.
    AVIF_DECODER_SOURCE_AUTO = 0,

    ...

    // Use the chunks inside primary/aux tracks in the moov block.
    // This is where avifs image sequences store their images.
    AVIF_DECODER_SOURCE_TRACKS

    ...
} avifDecoderSource;
```

Reply:
Done.